src/Composer/ScriptHandler.php

```diff
@@ -69,4 +85,30 @@ function ($type, $buffer) use ($event) {
             throw new \RuntimeException('An error occurred while executing the "' . $cmd . '" command.');
         }
     }
+
+    /**
+     * Validates that we can generate a random secret.
+     * We need to make sure at least one file is defined in the config but none of the files exist.
+     *
+     * @param array $config
+     *
+     * @return bool
+     */
+    private static function canGenerateSecret(array $config)
+    {
+        // "incenteev-parameters" config can be an array of files.
+        if (!isset($config['file'])) {
+            $result = false;
+
+            foreach ($config as $v) {
+                if (is_array($v) && isset($v['file']) && !is_file($v['file'])) {
+                    $result = true;
```

Comment:
Because there could be a second file that does exist (and then we must not set the variable)